### PR TITLE
Add search (filter) for an specific Integration on Homepage's Integrations section 

### LINKED ISSUE #1798: Add search (filter) for an specific Integration on Homepage's Integrations section
### Current Behavior

On https://meshery.io/ homepage  a section displays all the integrations that the project supports.
### Desired Behavior

[meshery.io](http://meshery.io/)’s home page is in need of a search (filter) for site visitors who wish to search for a specific integration.<br>

### Screenshots / Mockups

Screenshot 2024-07-03 at 2 19 02 PM
---
<h4>Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources</h4>

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by shubhusion**
**Description**

This PR fixes #1798

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@hargunkaur286 you might offer review.

**Review by hargunkaur286 — COMMENTED** [review]
@shubhusion Can you share screenshots of the changes?
The images are missing in the deploy preview
Screenshot 2024-07-31 at 4 29 54 PM

**Comment by shubhusion** [comment]
> @shubhusion Can you share screenshots of the changes? The images are missing in the deploy preview Screenshot 2024-07-31 at 4 29 54 PM

@hargunkaur286 I have only added the search filter present above the images 

image

**Comment by hargunkaur286** [comment]
> > @shubhusion Can you share screenshots of the changes? The images are missing in the deploy preview Screenshot 2024-07-31 at 4 29 54 PM
> 
> @hargunkaur286 I have only added the search filter present above the images
> 
> image

okay, can you share a full screenshot please

File: _includes/partials/homePage-integrations.html [hunk 1]
```diff
@@ -1,83 +1,217 @@
...
+                .slice(0, cardsToShow)
+                .map(
+                    (item) => `
+            <a href="${item.permalink}" class="integrations-card">
+              <div class="integrations-card-inner">
                 <div class="card-front">
-                <img src="${iconURL}" loading="lazy" id="logo-dark-light"
-                data-logo-for-dark="${iconURL}"
-                data-logo-for-light="${iconURL}" />
+                  <img src="${item.color}" loading="lazy" id="logo-dark-light"
```
Comment (hargunkaur286) [inline _includes/partials/homePage-integrations.html:158]: I see the error, you have to call the icon's url here in the image tag, how are you calling 'item.color' in  tag? this is causing the issue of images not getting displayed.

**Review by hargunkaur286 — CHANGES_REQUESTED** [review]

**Comment by shubhusion** [comment]
Hi @hargunkaur286,

I have made the necessary changes. However, while rebasing, I commited an error that caused the commits of other collaborators to merge into this PR. So, I have closed this PR and created a new one.

Could you please review the new PR? Thanks!

PR: https://github.com/meshery/meshery.io/pull/1837

**Pull request closed without merging**